documentation/RLC-quickstart.md

```diff
@@ -465,7 +465,13 @@ In order to release it, we need to add some tests for it to make sure we are del
     ```
 # How to write samples
 
-There're samples for TypeScript and JavaScript and for dev, You may copy the [samples folder](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-account-rest/samples) and [samples-dev folder](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-account-rest/samples-dev) and then change everything into your own services, including package-name, sample code, readme description etc.
+There're samples for TypeScript and JavaScript and for dev, You may copy the [samples-dev folder](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-account-rest/samples-dev) and then change everything into your own services, including package-name, sample code, readme description etc. Then we provide tools to automatically change it into workable samples in both TypeScript and JavaScript. The steps are:  
```

Comment:
Do you have any guide on how to run the sample? Is it possible to provide package.json file as part of the sample?

Reply:
Is it also better to leverage a code template instead of the code for a specific SDK for reference?

Reply:
`dev-tool samples publish -f` will generate everything, like package.json tsconfig.json readme.md etc.

Reply:
For running the samples you need to execute prom the project root

```sheel
dev-tool samples run samples-dev
```

@willmtemple when first creating a new SDK, do we need to run `dev-tool samples publish`? Or is publish handled automatically by the post-release bot?

Reply:
I would also mention tsconfig.json has to include the following under compilerOptions

```json
    "paths": {
      "@azure-rest/{servicename}-{modulename}": ["./src/index"]
    }
```

This is what wires the samples with the SDK so that we can do `import FooClient from "@azure-rest/foo"` instead of  `import FooClient from "../src"`